Core/HLE/sceNp2.cpp

```diff
@@ -0,0 +1,566 @@
+// Copyright (c) 2012- PPSSPP Project.
```

Comment:
nit: New files should ideally use current year (i.e. `2022-`.)  I don't know of any locality where this would actually matter, though, but I'm not a licensed law person...

-[Unknown]